homeassistant/helpers/check_config.py

```diff
@@ -217,13 +233,13 @@ def _comp_error(ex: Exception, domain: str, config: ConfigType) -> None:
                 platform = p_integration.get_platform(domain)
             except loader.IntegrationNotFound as ex:
                 if not hass.config.recovery_mode and not hass.config.safe_mode:
-                    result.add_error(f"Platform error {domain}.{p_name} - {ex}")
+                    result.add_warning(f"Platform error {domain}.{p_name} - {ex}")
```

Comment:
We should maybe still report this in recovery and safe mode, but with a different error message?

Reply:
Of maybe just change the "log level" to debug ie the color in the script output?

Reply:
Yes, that could also work. I'd like to address this in a follow-up though.